.github/scripts/run-tests.sh

```diff
@@ -56,6 +56,9 @@ case $1 in
     chipyard-hetero)
         run_bmark ${mapping[$1]}
         ;;
+    chipyard-prefetchers)
+        make -C $LOCAL_SIM_DIR $DISABLE_SIM_PREREQ ${mapping[$1]} run-binary BINARY=$RISCV/riscv64-unknown-elf/share/riscv-tests/benchmarks/dhrystone.riscv
```

Comment:
Side note: Why don't we make LOADMEM the default? Isn't it fairly robust now?

Reply:
LOADMEM only works on configs that rely on the normal backing memory model. For configs which rely on on-chip memory, or which have a more sophisticated memory system, setting LOADMEM may result in a silent failure that is difficult to debug.